### PR TITLE
[doc]Added section in 'How to' in Getting-Started.md

### PR DESCRIPTION
**Pull request by ria18405**
Signed-off-by: Ria Gupta <ria18405@iiitd.ac.in>

### CODE REVIEW

File: Getting-Started.md [hunk 1]
```diff
@@ -499,3 +499,13 @@ Alternatively, a data table visualization can be created using the steps mention
 Kibiter then automatically generates a query as shown below.
 
 ![Creating data query through request](https://user-images.githubusercontent.com/32506591/77188682-5e065e80-6afc-11ea-9b00-a927690cc76e.png)
+
+#### Remove existing dockers and start a fresh environment [&uarr;](#how-to-)
+
+   - Ensure that there is no entry in `docker ps` output. To do that, run `docker-compose down --remove-orphans`
```
Comment (valeriocos) [inline Getting-Started.md]: What do you think about listing each step? Something similar to:

- Stop and remove the containers with `docker-compose down --remove-orphans`
- Ensure that there is no entry in `docker ps`
- Remove all unused containers, images, and volumes with ` docker system prune -a --volumes`
- Execute `docker-compose up -d` (see https://github.com/chaoss/grimoirelab-sirmordred/blob/master/Getting-Started.md#source-code-and-docker-)
- Check connection to Elasticsearch with `curl -XGET elasticsearch-url -k`. The output should be the similar to this one
```
{
  "name" : "2PnErid",
  "cluster_name" : "bitergia_elasticsearch",
  "cluster_uuid" : "9RdkRSc7Tla8GlRVm7l4uA",
  "version" : {
    "number" : "6.1.0",
    "build_hash" : "c0c1ba0",
    "build_date" : "2017-12-12T12:32:54.550Z",
    "build_snapshot" : false,
    "lucene_version" : "7.1.0",
    "minimum_wire_compatibility_version" : "5.6.0",
    "minimum_index_compatibility_version" : "5.0.0"
  },
  "tagline" : "You Know, for Search"
}
```
- Execute micro mordred (https://github.com/chaoss/grimoirelab-sirmordred#micro-mordred-)

Comment (ria18405) [reply Getting-Started.md]: Hi @valeriocos , Thanks for reviewing!
Yes right, listing this way makes more sense. I'll change it.

Thanks :)

Comment (ria18405) [reply Getting-Started.md]: @valeriocos 
I have made the required changes. 

Thanks

File: Getting-Started.md [hunk 2]
```diff
@@ -499,3 +499,36 @@ Alternatively, a data table visualization can be created using the steps mention
...
 
 ![Creating data query through request](https://user-images.githubusercontent.com/32506591/77188682-5e065e80-6afc-11ea-9b00-a927690cc76e.png)
+
+#### Remove existing dockers and start a fresh environment [&uarr;](#how-to-)
+
+   - Stop and remove all the containers with `docker-compose down --remove-orphans`.
+   
+   - Run `docker ps` and ensure that there is no entry in the output.
+    
+   - Remove all unused containers, images, and volumes with  `docker system prune -a --volume`.
```
Comment (valeriocos) [inline Getting-Started.md]: --volume or --volumes?

**Comment by ria18405** [comment]
@valeriocos ,
I'm really sorry about this error. I somehow missed it. I'll correct it right away.

Thanks for pointing out.

**Comment by vchrombie** [comment]
Hi @ria18405 
Great work! 
The PR seems to be good.

I just have one small comment. It seems that you forgot to remove the other commit messages while squashing. It would be good if you can remove the extra commit messages and extra signoffs from the commit description. Just to maintain good commit history.
You can simply edit the commit message using `git commit --amend` and force push the changes.
Thanks. :slightly_smiling_face:

**Comment by ria18405** [comment]
Thanks @vchrombie

**Pull request merged**